src/DssSpell.t.base.sol

```diff
@@ -1693,6 +1693,35 @@ contract DssSpellTestBase is DSTest, DSMath {
             calc_step:    120,
             calc_cut:     9990
         });
+        afterSpell.collaterals["GUNIV3DAIUSDC2-A"] = CollateralValues({
+            aL_enabled:   true,
+            aL_line:      10 * MILLION,
+            aL_gap:       10 * MILLION,
+            aL_ttl:       8 hours,
+            line:         10 * MILLION,
```

Comment:
this should be set to zero as Autoline is enabled.
you can see for reference the following comment in `ETH-A`:
`// Not checked here as there is auto line`
```suggestion
            line:         0,
```